addons/website/views/snippets/snippets.xml

```diff
@@ -87,6 +87,9 @@
                 <t t-snippet="website.s_three_columns" string="Columns" group="columns">
                     <keywords>columns, description</keywords>
                 </t>
+                <t t-snippet="website.s_card_offset" string="Card Offset" group="columns">
```

Comment:
@xlu-odoo  Don't forget to rebase your branch and move the declaration of your snippet to the right place 👍